tools/docker/builder/openwrt/scripts/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+sed -ri "s/-DTARGET_PLATFORM_TYPE=[^ ]*/-DTARGET_PLATFORM_TYPE=${TARGET_PLATFORM_TYPE}/" feeds/prpl/prplmesh/Makefile
+sed -ri "s/PKG_REV:=.*$/PKG_REV:=${PRPLMESH_VERSION}/" feeds/prpl/prplmesh/Makefile
+make V=sc
+find bin -name 'prplmesh*.ipk' -exec cp {} "prplmesh-${TARGET}-${TARGET_PLATFORM_TYPE}-${OPENWRT_VERSION}-${PRPLMESH_VERSION}.ipk" \;
```

Comment:
Is there a chance that the find will match several ipk files?

Reply:
In theory yes, if another package builds an ipk matching "prplmesh*".
I don't see any easy way to get an exact match at this point though. 
We could check that the version number is following the package name maybe.
For reference, here is the full package name for the rax40:  `prplmesh_1.0-1_mips_24kc_nomips16.ipk`.